os/arch/arm/src/s5j/sss/security_hal_wrapper.c

```diff
@@ -0,0 +1,1627 @@
+/****************************************************************************
```

Comment:
And it using constant many values but almost no description.
Then it is very hard to understand when other ppl see yours.